lib/multierror/multierror.go

```diff
@@ -46,13 +59,16 @@ func NewOr(errs []error, fallback error) error {
 // Unwrap for MultiError always returns nil, as there is no reasonable way to implement it.
 //
 // Use As and Is methods, or loop over the list directly to access the underlying errors.
-func (me MultiError) Unwrap() error {
-	return nil
+func (multi MultiError) Unwrap() error {
+	if len(multi) == 0 {
+		return nil
+	}
+	return multi[0]
 }
 
 // As for MultiError returns the first error that can be considered As the specified target.
-func (me MultiError) As(target interface{}) bool {
-	for _, err := range me {
+func (multi MultiError) As(target interface{}) bool {
```

Comment:
One letter should be sufficient; `me` was fine - why the change to `multi`?

Reply:
once we start linting it would come up as too generic. Superficial change.